docs/_adapters/nginx-sm.md

```diff
@@ -12,8 +12,7 @@ permalink: service-meshes/adapters/nginx-sm
 ---
 {% include adapter-status.html %}
 
-The {{ page.name }} is currently under construction ({{ page.project_status }} state). Want to contribute? Check our [progress](page.github_link).
-
+The {{ page.name }} is currently under construction ({{ page.project_status }} state). Want to contribute? Check our [progress](https://github.com/layer5io/meshery-nginx-sm).
```

Comment:
Let's use `{{ page.github_link }}` instead.

Reply:
Ditto as to what @Nikhil-Ladha said.

Reply:
Haa! i was 6 seconds late😂

Reply:
Minus the round-trip latency from GitHub's server (wherever that is), and we're tied. 😄

Reply:
@leecalcote I tried that but if that is used I wasn't sure if the link would be attached to this text
As the URLs in Markdown required `[ ]( ) --> This Format`
![image](https://user-images.githubusercontent.com/23498248/115253072-a94ccf80-a149-11eb-8a77-df7f1c10feaf.png)

Reply:
But I've fixed that in the latest comment :)